Lib/Rabbit4000/tcpip/DMAETH100.LIB

```diff
@@ -808,34 +817,33 @@ dma_rcv_isr::
    jr    nz,.reset_rx_port ; 5
 
    ; Check if only status rcv'd with no packet data
+   ; This is the catch all for errors with R6000
 	ld		hl,(ix+[_decb]+rcvlen)
    test	hl
    jr		z,.test_rx_error
 
    ; Check for valid Rcv Packet
-	ld		a,(ix+[_decb]+rcvlsbstat)
+   ; Don't know where this is documented for R6000?

```

Comment:
That's scary to me.  If there was some blurb in the R6000 manual that referenced the R5000 (or even the R4000), then fine.  I just can't believe the manual would detail operation and then leave this part out???